faiss_vector_test.go

```diff
@@ -335,6 +344,43 @@ func letsCreateVectorIndexForTesting(dataset [][]float32, dims int, similarity s
 	return idx, nil
 }
 
+func TestRemoveVectorIDs(t *testing.T) {
+
+	testCases := []struct {
+		numVecs int
+	}{
+		{
+			// 5 vectors to test for flat indexes.
+			numVecs: 5,
```

Comment:
TODO Find a way to ensure all the types of vec indexes are tested in such UTs without using hard limits like 5/500 which may be prone to staleness if the code is updated.